_docs/proxying.md

```diff
@@ -158,6 +158,8 @@ stubFor(get(urlEqualTo("/users/12345.json"))
   .withStatus(503)));
 ```
 
+Also, we can enable/disable pass through unmatched requests to the target indicated by the original requests by enabling/disabling proxyPassThrough flag.  This flag can be set either by passing option while running standalone jar or by updating global settings through API.
```

Comment:
Please can we have a link to the CLI parameter here?

And also an example in both Java and JSON of how to set the flag via a global call?

Reply:
sure, incorporated